src/webui/web_server_utils/terminal_mixin.py

```
@@ -0,0 +1,139 @@
+        Args:
+            command: Shell command text to execute.
+
+        Returns:
+            Tuple of response payload and HTTP status code.
+        """
+        cwd_file = tempfile.NamedTemporaryFile(
+            mode="w", prefix="eagleeye-terminal-", suffix=".cwd", delete=False
+        )
+        cwd_file.close()
+        try:
+            completed_process = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)